html-forms/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+<h1>Coding Form!</h1>
+<form action="#">
+
+  <div>
+    <label>Email:
+    <br>
+    <input type="email" id="email">
+    </label>
+
+    <br>
```

Comment:
instead of the `br` tags, I would recommend using `div` tags to segment out the label/inputs onto a new line